driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1593,7 +1593,11 @@ boolean removeSession(Session session) {
 
         void reportQuery(Host host, Statement statement, Exception exception, long latencyNanos) {
             for (LatencyTracker tracker : latencyTrackers) {
-                tracker.update(host, statement, exception, latencyNanos);
+                try {
```

Comment:
Not really related, but I thought it would be wise to catch exceptions here instead of letting them propagate.